_posts/2015-04-28-how-I-found-the-cheapest-place-to-live-in-austin.md

```diff
@@ -0,0 +1,274 @@
+
+I didn't have too much of a plan when I arrived. I had met a guy on 
+[Hashtag Nomads](hashtagnomads.com) who had offered me to stay in his couch for 
+free. I had accepted that offer because I wanted to kick-start my trip into 
+Austin, and get immersed in the culture and local knowledge as soon as possible 
+was one of the priorities. So off I went and said yes to Randall, the stranger 
+I had met on the Internet. Oh, and before you tell me that I'm crazy, Randall 
+had sent me a video of his adventures while he **BIKED FROM GERMANY TO INDIA**, 
+which convinced me that he was not, in fact, an axe murderer. I confirm with him
+a couple of days before my flight, and off we go. What could possibly go wrong?
+
+As soon as I landed, I gave him a call:
+
+> Hey man, how's it going? Max here! I told you on Hashtag Nomads that I would
+> come and crash at your place, so here I am! Where do we meet?
+
+> Hey Max! Maaan, I completely forgot about you, I'm sorry. I'm actually in 
+> **Florida** right now for a bachelor party. But hey, just talk to my friend 
+> Albert, he's currently in town and might be able to help you out! I'll come 
+> back on Sunday [it was a Friday], and I can host you as soon as I arrive.
+
+Wonderful. It's starting to get dark around here, so I resort to call Albert
+right away:
+
+> Hey Albert, this is Max. Your friend Randall gave me your phone number
+> because...well, he's in Florida, and I was supposed to stay with him. What 
+> should I do?
+
+> Hey Max, um...I guess you'll have to rent an Airbnb. Austin's a safe place.
+> If you want, we can grab dinner today! I can drop you off afterwards wherever 
+> you end up staying at. You up? And good luck!
+
+> Sure, sure. I'm down. _Let's do this._
+
+It is official now. It's 6pm and I don't have a place to sleep. So I grab my
+phone, download Airbnb (a life saver), find the cheapest listing available and 
+message Julia, the host. She gets back to me within 5 minutes, reassuring me 
+that the house door uses a passcode lock, which enables her to just send the 
+code and I'm in. Long story short, I book the place for the two nights, while 
+Randall comes back. I now have a place to stay. I love technology.
+
+And there was evening, and there was morning — the first day.
+
+## Randall
+
+The Airbnb I stayed at was fantastic. It was a DIY solution; I didn't get
+to meet the host until the next day. The front door, like I said, could be
+opened with a passcode, and the house was equipped with 6 rooms, each with their 
+own passcode. Julia gave each guest the code for the front door, and the one to 
+their own rooms. My bed was clean, towels were readily available, and most 
+importantly, the WiFi password was on a piece of paper stuck to the fridge.
+
+I spent the next two days exploring around, going to coffee shops nearby and
+working. One of the benefits of working remotely, of course, is that you get to
```

Comment:
One of the benefits of working remotely, of course, is that you _don't even skip a beat at work_ while being half-stranded in an Airbnb somewhere in Austin.